CONTRIBUTING.md

```diff
@@ -34,10 +35,9 @@ and Python log messages in the debug console under "Python Server".
 - Update the `ruff` version in the README.md
   - in the Base URLs
   - in "The extension ships with `ruff==...`"
-- Make sure you use Python 3.7 installed and as your default Python.
-- Run `python -m venv .venv` to create a venv and activate it.
-- Run `python -m pip install pip-tools` to install `pip-tools`.
-- Run `rm requirements.txt requirements-dev.txt` and then `just lock` to update `ruff` and `ruff-lsp`.
+- Make sure you have Python 3.7 installed
+- Run `uv venv --python 3.7 && source .venv/bin/activate` to create a Python 3.7 venv and activate it.
```

Comment:
Unfortunately `uv` will not install Python 3.7 for you if you don't have it already installed; 3.7 is too old for `uv`

Reply:
Ah yeah, I remember facing this

Reply:
I think Python is only required to bundle `ruff-lsp` so it should be fine

Reply:
Python 3.7 is also still installable by other means, so it's "okay", just makes it a bit more annoying to work with if you don't already have Python 3.7 installed locally. Luckily I do 😄